src/data/set/opposite.lean

```diff
@@ -9,6 +9,10 @@ import data.set.image
 /-!
 # The opposite of a set
 
+> THIS FILE IS SYNCHRONIZED WITH MATHLIB4.
+> https://github.com/leanprover-community/mathlib4/pull/None
```

Comment:
```suggestion
> https://github.com/leanprover-community/mathlib4/pull/983
```

Reply:
This means that something is wrong with the yaml